src/main/java/com/github/soerxpso/xpspawners/listeners/MobSpawnListener.java

```diff
@@ -21,13 +20,12 @@ public MobSpawnListener() {
 	@EventHandler(priority = EventPriority.LOW, ignoreCancelled = false)
 	public void onMobSpawn(SpawnerSpawnEvent e) {
 		e.setCancelled(true);
-		e.getSpawner().setDelay(ConfigManager.getHarvestInterval());
+		e.getSpawner().setDelay(Integer.MAX_VALUE);
```

Comment:
Oops. The Spawner class got unadded somehow. I fixed it.

There's now a Bukkit TimerTask in the Spawner class that schedules drops. Using the SpawnerSpawnEvent without any sort of timer was causing occasional double drops.